src/main/java/taub/gameoflife/GameOfLife.java

```diff
@@ -0,0 +1,82 @@
+package taub.gameoflife;
+
+import java.util.Arrays;
+
+public class GameOfLife
+{
+    private int[][] origGameBoard;
+    private int[][] newGameBoard;
+
+    public GameOfLife(int[][] board)
+    {
+        origGameBoard = board;
+        setNewGameBoard();
+    }
+
+    private void setNewGameBoard()
+    {
+        newGameBoard = new int[origGameBoard.length][origGameBoard[0].length];
+        for (int i = 0; i < origGameBoard.length; i++)
+        {
+            newGameBoard[i] = origGameBoard[i].clone();
```

Comment:
calling clone is not necessary

Reply:
If I take out .clone(), then it doesn't keep the arrays separate and my array returns like this: [0, 1, 1]
                            [1, 0, 1]
                            [0, 0, 0]

Reply:
Nevermind